.gitignore

```diff
@@ -118,3 +118,5 @@ docbuild/
 
 # OSX directory info files
 .DS_Store
+docs/api_reference/index.rst
+examples/filename.png
```

Comment:
#1179 added `filename.png` to `.gitignore`, so it's probably worth merging with main and removing the duplicate.

nit: These should probably in a different section since they're not related to OSX.

Reply:
Not sure why the index file also ended up in gitignore, but I have removed them both.